.gitignore

```diff
@@ -21,9 +21,12 @@ hs_err_pid*
 *.iml
 
 ## Directory-based project format:
-.idea/
+.idea/*
```

Comment:
For me Idea doesn't really handle that correctly. Idea considers the files that are located in this DIR as git-files. So when I change something in one of these files - the file then is put to the list of files to commit ("Local changes"). Sometimes after some time Idea marks modified file as none-repo file but then again detects the changes. :(
@adyach